### DISCUSSION TITLE
Incorrect price on the website page

### DISCUSSION THREAD

**Discussion by kate0037** [streetwriters/notesnook#6884] [General]
![1](https://github.com/user-attachments/assets/cb8cf6b3-c6cc-44b9-a2fe-1a226cb9091d)
![2](https://github.com/user-attachments/assets/c70468d7-4005-4824-9d4c-bffc49c56a8d)

As you can see, paying monthly is cheaper than paying annually

**Comment by xa4hf8** [comment]
PostImage is not good for long-term hosting, they have already deleted your screen shots.  For future reference, you can just drag images from your desktop into the message composition frame here, but do make some effort to resize them if they seem unreasonably large in Preview mode.

> **_As you can see, paying monthly is cheaper than paying annually_**

It appears to me that the cost of a **Pro** subscription is `$4.49` per month if you purchase 1 month of service and `$4.17` per month if you purchase 12 months.  So it seems correct, but I can see how the page might be superficially confusing because it is inconsistent in the way that it presents information:

The cost of everything is shown as **_price-per-month_**, except for the educational license, which is shown as **_price-per-year_**.  That is just bad design, and it makes no sense to use different scales on different license categories:  The **Education** section should include the _**price-per-month**_ of an educational license, and the Pro section should include the _**price-per-year**_ of a **Pro** license.  You were right to call this out, and it needs to be fixed.

**Reply by kate0037** [reply]
Thanks for the reminder. I have reuploaded the images; it seems the incorrect price is due to regional pricing.
